doc/cephfs/fs-nfs-exports.rst

```diff
@@ -27,6 +27,15 @@ For more details on placement specification refer the `orchestrator doc
 <https://docs.ceph.com/docs/master/mgr/orchestrator/#placement-specification>`_.
 Currently only CephFS export type is supported.
 
+Update NFS Ganesha Cluster
+==========================
+
+.. code:: bash
+
+    $ ceph nfs cluster update <type=cephfs> <clusterid> <placement>
```

Comment:
Why do I need to specify type=cephfs here? Once the clusterid is created it should be unique. I don't see how that qualifier helps.

...or do you forsee this allowing you to change the type of an existing cluster? That seems fraught with peril if so.

Reply:
Yes, you are right. I will remove them.